generator/osm2meta.cpp

```diff
@@ -102,20 +102,8 @@ std::string MetadataTagProcessorImpl::ValidateAndFormat_stars(std::string const
 
 std::string MetadataTagProcessorImpl::ValidateAndFormat_operator(std::string const & v) const
 {
-  auto const & t = m_params.m_types;
-  if (ftypes::IsATMChecker::Instance()(t) ||
-      ftypes::IsPaymentTerminalChecker::Instance()(t) ||
-      ftypes::IsMoneyExchangeChecker::Instance()(t) ||
-      ftypes::IsFuelStationChecker::Instance()(t) ||
-      ftypes::IsRecyclingCentreChecker::Instance()(t) ||
-      ftypes::IsPostPoiChecker::Instance()(t) ||
-      ftypes::IsCarSharingChecker::Instance()(t) ||
-      ftypes::IsCarRentalChecker::Instance()(t) ||
-      ftypes::IsBicycleRentalChecker::Instance()(t) ||
-      ftypes::IsParkingChecker::Instance()(t))
-  {
+  if (ftypes::IsPoiChecker::Instance()(m_params.m_types))
```

Comment:
Hmm, maybe there was a reason "operator" was picked up for certain POIs only - probably only for those where it should highly likely make sense.

E.g. a particular restaurant in a franchise chain could be operated by some small/unknown company. See https://wiki.openstreetmap.org/wiki/Key:operator#Examples

Reply:
It is a follow-up from [here](https://github.com/organicmaps/organicmaps/pull/8523) and [here](https://github.com/organicmaps/organicmaps/issues/8039)

You can open a discussion there :)

Reply:
I did :)

Reply:
I also have mixed feelings about widely used 'operator' and 'network' tags, but POIs look like a reasonable tradeoff.

Reply:
What are the typical values in some well-mapped city? Let's see if they're useful for POIs.